ui/app/src/components/PathNavigator/PathNavigator.tsx

```diff
@@ -167,6 +168,12 @@ export default function PathNavigator(props: PathNavigatorProps) {
   const siteLocales = useSiteLocales();
   const hasActiveSession = useSelection((state) => state.auth.active);
 
+  useEffect(() => {
+    if (nnou(state?.keyword)) {
+      setKeyword(state.keyword);
```

Comment:
Could we use state.keyword and not have a duplicate local state keyword var?

Reply:
no at all, we need to keep one keyword value updated(the one the user should see in the pathNav) and another one debounced( the one is sent to the epics to fetch the data)